atlp-rwanda/rca-barefoot-team3-fe
# Setup react-router (React-router-dom)- #184637938

#### What does this PR do?
It configures the react-router-dom
#### Description of Task to be completed?
1. added the react-router-dom package
2. Configured the "/" page to test the configuration.
#### How should this be manually tested?
1. Clone the project
2. run `yarn` to install project packages
3. run `yarn dev` to start the project
4. check if you get heading _("Login Page")_ on the '/' page
#### What are the relevant pivotal tracker/Trello stories?
Setup react-router  (React-router-dom) - #184637938
